crates/bevy_pbr/src/material.rs

```diff
@@ -3,12 +3,22 @@ use bevy_reflect::TypeUuid;
 use bevy_render::{color::Color, renderer::RenderResources, shader::ShaderDefs, texture::Texture};
 
 /// A material with "standard" properties used in PBR lighting
+/// Standard property values with pictures here https://google.github.io/filament/Material%20Properties.pdf
 #[derive(Debug, RenderResources, ShaderDefs, TypeUuid)]
 #[uuid = "dace545e-4bc6-4595-a79d-c224fc694975"]
 pub struct StandardMaterial {
-    pub albedo: Color,
+    /// Doubles as diffuse albedo for non-metallic, specular for metallic and a mix for everything in between
+    pub base_color_factor: Color,
     #[shader_def]
-    pub albedo_texture: Option<Handle<Texture>>,
+    pub base_color_texture: Option<Handle<Texture>>,
+    /// Linear perceptual roughness, clamped to [0.089, 1.0] in the shader
+    /// Defaults to minimum of 0.089
+    pub roughness_factor: PerceptualRoughness,
+    /// From [0.0, 1.0], dielectric to pure metallic
+    pub metallic_factor: f32,
+    /// Specular intensity for non-metals on a linear scale of [0.0, 1.0]
+    /// defaults to 0.5 which is mapped to 4% reflectance in the shader
+    pub reflectance: f32,
```

Comment:
We should consider having `roughness_range`, `metallic_range`, and `reflectance_range` instead of plain f32 values. From my experience at Rust Ltd., my team's artist would often use [0,1] textures and then use the related factor to effectively set the range [0,factor] in order to reduce DXTC artifacts. So it could work even better if they could set a range instead and then just do `mix(range_min, range_max, texture_value)`. Though in the absence of a texture `range_min` would be ignored, so that might feel a little clunky.

Reply:
Makes total sense. But would you be OK with delaying these until the next PR? I was hoping to get this merged with just the material parameters and adding the maps in the next one.

Reply:
Sure. Sorry, just have a lot of ideas pouring out.

Reply:
Great!